scripts/ios-configure-glog.sh

```diff
@@ -50,8 +50,8 @@ if [ -h "test-driver" ]; then
 fi
 
 # Manually disable gflags include to fix issue https://github.com/facebook/react-native/issues/28446
-sed -i '' 's/\@ac_cv_have_libgflags\@/0/' src/glog/logging.h.in
-sed -i '' 's/HAVE_LIB_GFLAGS/HAVE_LIB_GFLAGS_DISABLED/' src/config.h.in
```

Comment:
This is needed as you're not using Mac OS `sed` but you replaced it with GNU `sed`.
If we merge this, I have the suspect this is going to break for the majority of OSX users: https://unix.stackexchange.com/a/131940

Reply:
Ah I see! In that case, let me change it slightly to support both instead, one moment...

Reply:
Updated! please let me know what you think

Reply:
Could you provide some refence on where the `-i.bak` is defined?

Reply:
In both the [gnu sed](https://linux.die.net/man/1/sed) and [macOS sed](https://www.unix.com/man-page/osx/1/sed/) `-i` takes an extension as an optional parameter for creating a backup file, however if it is not passed in it reads '-e' as the extension instead. `.bak` in particular is an arbitrary choice but makes the backup extension explicit especially for the subsequent deletion.

Reply:
@cortinico is this what you were looking for? or did you mean something different?

Reply:
Yup thanks for clarifying 👍

Reply:
@cortinico I see that the PR failed lint tests, but I can't access the tests to see what's wrong. If you have access to these would you mind helping me find out what's causing it to fail?

Reply:
@alphashuro I'm taking care of this right now. The tab literal characters in the patch content is making our internal linters angry. Stand by :)